vc-verifier/kotlin/vcverifier/publish-artifact.gradle

```diff
@@ -0,0 +1,165 @@
+publishing {
+    repositories {
+        maven {
+            url = System.getenv("OSSRH_URL")
+            name = "vcverifier"
+            credentials {
+                username = System.getenv("OSSRH_USER")
+                password = System.getenv("OSSRH_SECRET")
+            }
+        }
+    }
+    publications {
+        aar(MavenPublication) {
+            pom {
+                withXml {
+                    def dependenciesNode = asNode().appendNode('dependencies')
```

Comment:
Please refer pixelpass to add all mandatory nodes and signing for release build

Reply:
Updated